src/instana/instrumentation/couchbase_inst.py

```diff
@@ -6,90 +6,148 @@
 https://docs.couchbase.com/python-sdk/2.5/start-using-sdk.html
 """
 
+from typing import Any, Callable, Dict, Tuple, Union
+
 import wrapt
 
-from ..log import logger
-from ..util.traceutils import get_tracer_tuple, tracing_is_off
+from instana.log import logger
+from instana.span.span import InstanaSpan
+from instana.util.traceutils import get_tracer_tuple, tracing_is_off
 
 try:
     import couchbase
-
-    if not (hasattr(couchbase, '__version__') and couchbase.__version__[0] == '2'
-            and (couchbase.__version__[2] > '3'
-                 or (couchbase.__version__[2] == '3' and couchbase.__version__[4] >= '4'))
-        ):
+    from couchbase.bucket import Bucket
+
+    if not (
+        hasattr(couchbase, "__version__")
+        and couchbase.__version__[0] == "2"
+        and (
+            couchbase.__version__[2] > "3"
+            or (couchbase.__version__[2] == "3" and couchbase.__version__[4] >= "4")
+        )
+    ):
```

Comment:
Can we make it simpler?

```suggestion
if not hasattr(couchbase, "__version__") and (couchbase.__version__ < "2.3.4" or couchbase.__version__ >= "3.0.0"):
```